test/integ/pandas/test_pandas_tools.py

```diff
@@ -809,20 +829,8 @@ def test_all_pandas_types(
             assert nchunks == 1
             # Check table's contents
             result = cnx.cursor(DictCursor).execute(select_sql).fetchall()
-            for row in result:
-                assert (
-                    row["int"],
-                    row["float"],
-                    row["string"],
-                    row["bool"],
-                ) in df_data_no_timestamps
-                # TODO: Schema detection on the server-side has bugs dealing with timestamp_ntz and timestamp_tz.
-                #  After the bugs are fixed, change the assertion to `data[0]["tm_tz"] == datetime_with_tz`
-                #  and `data[0]["tm_ntz"] == datetime_with_ntz`,
-                #  JIRA https://snowflakecomputing.atlassian.net/browse/SNOW-524865
-                #  JIRA https://snowflakecomputing.atlassian.net/browse/SNOW-359205
-                #  JIRA https://snowflakecomputing.atlassian.net/browse/SNOW-507644
```

Comment:
Is this TODO being addressed as well?

Reply:
yes

Reply:
ok actually it's not solved, and we need to turn on a parameter to solve this.

Reply:
Then should we also keep the TODO?

Reply:
Yes, I added it back with updated jira